packages/app/src/systems/Asset/components/AssetsAmount/AssetsAmount.test.tsx

```diff
@@ -6,17 +6,23 @@ import { AssetsAmount } from './AssetsAmount';
 
 describe('AssetsAmount', () => {
   it('a11y', async () => {
-    await testA11y(<AssetsAmount amounts={MOCK_ASSETS_AMOUNTS} />);
+    await testA11y(
+      <AssetsAmount amounts={MOCK_ASSETS_AMOUNTS} title="Assets to Send" />
+    );
   });
 
   it('should show multiple assets', async () => {
-    render(<AssetsAmount amounts={MOCK_ASSETS_AMOUNTS} />);
+    render(
+      <AssetsAmount amounts={MOCK_ASSETS_AMOUNTS} title="Assets to Send" />
+    );
     expect(screen.getByText('Ethereum')).toBeInTheDocument();
     expect(screen.getByText('Dai')).toBeInTheDocument();
   });
 
   it('should show positive values with plus', async () => {
-    render(<AssetsAmount amounts={MOCK_ASSETS_AMOUNTS} />);
-    expect(screen.getByText('14563943.834 ETH')).toBeInTheDocument();
+    render(
+      <AssetsAmount amounts={MOCK_ASSETS_AMOUNTS} title="Assets to Send" />
+    );
+    expect(screen.getByText('14.5 ETH')).toBeInTheDocument();
```

Comment:
The test expected to show a + sign, but it didn't check it. Maybe change the value to include the +

Reply:
we have `isPositive` and `isNegative` to show signals. I will add tests to it

When there's only `Assets to send`, We don't show the `-` signal to avoid confusion.

It only makes sense when you have both `Assets to send` and `Assets to receive`, then we use the signals to differentiate them.